virtcontainers/pkg/cloud-hypervisor/client/api/openapi.yaml

```diff
@@ -309,24 +330,26 @@ components:
         devices:
         - path: path
           iommu: false
+          id: id
         - path: path
           iommu: false
+          id: id
         kernel:
           path: path
         rng:
           iommu: false
           src: /dev/urandom
         fs:
         - sock: sock
-          num_queues: 9
-          queue_size: 3
-          cache_size: 2
+          num_queues: 3
+          queue_size: 2
+          cache_size: 4
           dax: true
           tag: tag
         - sock: sock
-          num_queues: 9
-          queue_size: 3
-          cache_size: 2
+          num_queues: 3
+          queue_size: 2
+          cache_size: 4
```

Comment:
I don't understand all these numbers assigned to `num_queues` and `queue_size` :-/
Is this coming from the automated file generation?

Reply:
Yes, it is comming from the automatic file generation seems that it adds examples to that yaml. 

https://github.com/kata-containers/runtime/pull/2556/files#diff-4dc59e987e364456f339849060f41612L160

Reply:
Ok!